pkg/metrics/prometheus_exporter.go

```diff
@@ -16,7 +17,11 @@ var log = logf.Log.WithName("metrics")
 const namespace = "gatekeeper"
 
 func newPrometheusExporter() (view.Exporter, error) {
-	e, err := prometheus.NewExporter(prometheus.Options{Namespace: namespace})
+	e, err := prometheus.NewExporter(prometheus.Options{
+		Namespace:  namespace,
+		Registerer: ctlmetrics.Registry,
+		Gatherer:   ctlmetrics.Registry,
```

Comment:
Do we need to explicitly set `Gatherer` or would it just use `Registerer`?

Reply:
I believe they're both required - the `Registerer` is the interface for registering the available metrics, but the `Gatherer` is the interface called to collect all the metrics at scrape time. If the `Gatherer` was a different registry instance, it would have no knowledge of the registered metrics.